packages/cf-builder-card/src/CardBuilder.js

```diff
@@ -48,21 +66,7 @@ class CardBuilder extends React.Component {
   }
 }
 
-CardBuilder.propTypes = {
```

Comment:
You'd have to re-specify the types for `propTypes` `defaultProps` and all the `contextProps` stuff. Otherwise Flow wouldn't consider these actual members of the class.